cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/stats/BasicStats.java

```diff
@@ -71,6 +84,10 @@ int numLeaders() {
     return _numLeaders;
   }
 
+  double getDiskCapacity() {
```

Comment:
Nit: just a minor convention for `getters` we try to use in cc: `getDiskCapacity()` -> `diskCapacity()`